bundles/org.openhab.binding.easee/src/main/java/org/openhab/binding/easee/internal/EaseeBindingConstants.java

```diff
@@ -161,7 +161,7 @@ public class EaseeBindingConstants {
     public static final String THING_CONFIG_BACK_PLATE_ID = "backPlateId";
     public static final String THING_CONFIG_MASTER_BACK_PLATE_ID = "masterBackPlateId";
 
-    public static final Instant OUTDATED_DATE = Instant.MIN;
+    public static final Instant OUTDATED_DATE = Instant.EPOCH;
```

Comment:
Why do you change that ?
The previous value looks more appropriate, even if it will also work with EPOCH.

Reply:
just wanted to be sure but as I have added additional checks now it should of course be fine if I keep Instant.MIN